build/commands/lib/util.js

```diff
@@ -62,7 +62,7 @@ async function applyPatches() {
   devtoolsFrontendPatchStatus.forEach(
     s => s.path = path.join('third_party', 'devtools-frontend', 'src', s.path))
   const allPatchStatus = [...chromiumPatchStatus, ...v8PatchStatus, ...catapultPatchStatus, ...devtoolsFrontendPatchStatus, ...ffmpegPatchStatus]
-  Log.allPatchStatus(allPatchStatus, 'Chromium')
+  Log.allPatchStatus(allPatchStatus, 'Chromium', printPatchFailuresInJson)
```

Comment:
Perhaps we can create a separate function, e.g. `printPatchFailuresToJson`? If we don't need the successes logged to stdout then we could wrap in an `if (printPatchFailuresInJson) ... else ...`, otherwise we can do in addition. Just looking to avoid complicating `Log.allPatchStatus` if it's not necessary.

Reply:
Done.